> Should be `5.0.0.payara-p1`

it was needed to increment from 5.0.0 to 5.0.1 because of bnd validations on the project:
`
[INFO] The baseline version was found to be 5.0.0
[ERROR] The bundle version change (5.0.0 to 5.0.0) is too low, the new version must be at least 5.0.1
`

--- a/api/pom.xml
+++ b/api/pom.xml
@@ -19,12 +19,16 @@
     <parent>
         <groupId>org.eclipse.microprofile.metrics</groupId>
         <artifactId>microprofile-metrics-parent</artifactId>
-        <version>5.0.1.payara-p1</version>
+        <version>5.0.0.payara-p1</version>
     </parent>
 
     <artifactId>microprofile-metrics-api</artifactId>
     <name>MicroProfile Metrics API</name>
     <description>MicroProfile Metrics :: API</description>
+
+    <properties>
+        <bnd.baseline.continue.on.error>true</bnd.baseline.continue.on.error>
+    </properties>
 
     <dependencies>
         <dependency>

--- a/pom.xml
+++ b/pom.xml
@@ -24,7 +24,7 @@
 
     <groupId>org.eclipse.microprofile.metrics</groupId>
     <artifactId>microprofile-metrics-parent</artifactId>
-    <version>5.0.1.payara-p1</version>
+    <version>5.0.0.payara-p1</version>
     <packaging>pom</packaging>
     <name>MicroProfile Metrics</name>
     <description>Eclipse MicroProfile Metrics Feature :: Parent POM</description>

--- a/spec/pom.xml
+++ b/spec/pom.xml
@@ -20,7 +20,7 @@
         <!-- This is just for now and will not work if the API has a separate release cycle than the rest. -->
         <groupId>org.eclipse.microprofile.metrics</groupId>
         <artifactId>microprofile-metrics-parent</artifactId>
-        <version>5.0.1.payara-p1</version>
+        <version>5.0.0.payara-p1</version>
     </parent>
 
     <artifactId>microprofile-metrics-spec</artifactId>

--- a/tck/api/pom.xml
+++ b/tck/api/pom.xml
@@ -13,7 +13,7 @@
         <!-- This is just for now and will not work if the API has a separate release cycle than the rest. -->
         <groupId>org.eclipse.microprofile.metrics</groupId>
         <artifactId>microprofile-metrics-tck</artifactId>
-        <version>5.0.1.payara-p1</version>
+        <version>5.0.0.payara-p1</version>
         <relativePath>../pom.xml</relativePath>
     </parent>
 

--- a/tck/optional/pom.xml
+++ b/tck/optional/pom.xml
@@ -20,7 +20,7 @@
         <!-- This is just for now and will not work if the API has a separate release cycle than the rest. -->
         <groupId>org.eclipse.microprofile.metrics</groupId>
         <artifactId>microprofile-metrics-tck</artifactId>
-        <version>5.0.1.payara-p1</version>
+        <version>5.0.0.payara-p1</version>
         <relativePath>../pom.xml</relativePath>
     </parent>
 

--- a/tck/pom.xml
+++ b/tck/pom.xml
@@ -20,7 +20,7 @@
         <!-- This is just for now and will not work if the API has a separate release cycle than the rest. -->
         <groupId>org.eclipse.microprofile.metrics</groupId>
         <artifactId>microprofile-metrics-parent</artifactId>
-        <version>5.0.1.payara-p1</version>
+        <version>5.0.0.payara-p1</version>
 
         <relativePath>../pom.xml</relativePath>
     </parent>

--- a/tck/rest/pom.xml
+++ b/tck/rest/pom.xml
@@ -20,7 +20,7 @@
         <!-- This is just for now and will not work if the API has a separate release cycle than the rest. -->
         <groupId>org.eclipse.microprofile.metrics</groupId>
         <artifactId>microprofile-metrics-tck</artifactId>
-        <version>5.0.1.payara-p1</version>
+        <version>5.0.0.payara-p1</version>
         <relativePath>../pom.xml</relativePath>
     </parent>
 

